INSTALLING.md

```diff
@@ -2,199 +2,178 @@
 
 These instructions will guide you through the installation of Tenstorrent system tools and drivers, followed by the installation of TT-Metalium and TT-NN.
 
-> [!IMPORTANT]
->
-> If you are using a release version of this software, please check the installation instructions packaged with that version. You can find them in either the release assets for that version, or in the source files for that version tag.
-
 ---
+## Prerequisites:
 
-## Installation Steps
-
-### Step 1. Driver & Firmware
-
-Follow the Software Setup instructions for your specific board or system provided on our [general docs](https://docs.tenstorrent.com).
+### Step 1: [Setup the Hardware](https://docs.tenstorrent.com/quickstart.html#unboxing-and-hardware-setup)
 
-If you have purchased a Grayskull card, you will find the instructions [here](https://docs.tenstorrent.com/aibs/grayskull/installation.html).
+Once you have setup the hardware move on to step 2.
 
-Note the current compatability matrix:
+### Step 2: Install Driver & Firmware
 
-| Device              | OS              | Python   | Driver (TT-KMD)    | Firmware (TT-Flash)                        | TT-SMI                | TT-Topology                    |
-|---------------------|-----------------|----------|--------------------|--------------------------------------------|-----------------------|--------------------------------|
-| Grayskull           | Ubuntu 20.04    | 3.8.10   | v1.27.1            | fw_pack-80.9.0.0 (v80.9.0.0)               | v2.2.0 or above       | N/A                            |
-| Wormhole            | Ubuntu 20.04    | 3.8.10   | v1.27.1            | fw_pack-80.10.0.0 (v80.10.0.0)             | v2.2.0 or above       | N/A                            |
-| T3000 (Wormhole)    | Ubuntu 20.04    | 3.8.10   | v1.27.1            | fw_pack-80.10.0.0 (v80.10.0.0)             | v2.2.0 or above       | v1.1.3 or above, `mesh` config |
+#### Install System-level Dependencies
+```
+wget https://raw.githubusercontent.com/tenstorrent/tt-metal/refs/heads/main/install_dependencies.sh
```

Comment:
What’s the reason for using wget instead of just cloning? (Not saying it’s wrong, just want to understand).

Also, might need to do the following now:

chmod a+x install_dependencies.s

to make the script executable by root.

Reply:
this was an effort in simplifying the process (have fewer overall steps). The install_dependencies.sh also installs git and git-lfs which are necessary to cloning the repo. So the reasoning was providing an order of steps that provides every necessary component in order.

Reply:
Thank you @blozano-tt, bash was taking care of that but chmod seems like a safer option. Included it in latest commit.